point.py

```diff
@@ -0,0 +1,20 @@
+#from utils import check_coincident
+#import utils
+from .utils import *
```

Comment:
While `*` imports work, please do not use them.  They pollute the namespace in ways that can be hard to predict.